PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# My Awesome Pull Request
+
+## What does this change?
+
+## What does success look like and how do you plan to measure success?
+
+## Does this affect other platforms - Amp, Apps, etc?
```

Comment:
Possibly, but I think it's worth being prominent here as it's often forgot (I've fallen foul of it myself).